tox.ini

```
@@ -1,10 +1,17 @@
 [tox]
 envlist =
-  py39-django42-wagtail62
+  py{39,310,311,312}-django42-wagtail62
   py{310,311,312}-django{50,51}-wagtail62
   py312-django42-wagtail{52,60,61,62}
```

```suggestion
  py312-django42-wagtail{52,62}
```

since 6.0 and 6.1 are no longer officially supported

---

Ah, good call about Wagtail versions, thanks!

I'm not sure about only supporting Python 3.11+ though. In general, I agree with the sentiment, but I can imagine somebody jumping in to rescue a legacy project, and hoping to pull the production database, without doing a major upgrade beforehand. It's a simple package, and doesn't cost us much to support older Pythons (aside from some CI time).